src/IotTelemetrySimulator/DefaultRandomizer.cs

```diff
@@ -22,5 +22,18 @@ public int Next(int min, int max)
         {
             return this.generator.Value.Next(min, max);
         }
+
+        public double NextDouble()
```

Comment:
I would propose to generate a value between 0..1 if not boundaries have been defined.
This is going to generate any number.